bitnami/jaeger/values.yaml

```diff
@@ -1199,6 +1239,11 @@ cassandra:
   service:
     ports:
       cql: 9042
+  ## @param cassandra.initDB [object] Init script for initializing the instance
+  ##
+  initDB:
+    create_keyspace.cql: |
+      CREATE KEYSPACE {{ include "jaeger.cassandra.keyspace" . }} WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 };
```

Comment:
Nice! much simpler! Question: what happens if I'm using a external database? How's the db initialized?
```
externalDatabase:
  host: some-cassandra-host
  port: 9042
  dbUser:
    user: bn_jaeger
    password: some-password
cassandra:
  enabled: false
```

Reply:
The schema is now initialized by Jaeger.

Similar to other solutions that rely on Cassandra (e.g. zipkin), the keyspace needs to be created in the database for the application to connect.

Reply:
Added a note in the external database section just in case.